### ISSUE TITLE
NPE in idle notification plugin

### ISSUE THREAD

**Issue by deathbeam** [runelite/runelite#287]
This error is being thrown to console from time to time, probably when I get auto logged from the game and I am doing some action that is throwing idle notification (woodcutting in this case):

```
[Thread-7] WARN net.runelite.client.task.Scheduler - error invoking scheduled task
java.lang.reflect.InvocationTargetException
        at sun.reflect.GeneratedMethodAccessor23.invoke(Unknown Source)
        at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
        at java.lang.reflect.Method.invoke(Method.java:497)
        at net.runelite.client.task.Scheduler.run(Scheduler.java:100)
        at net.runelite.client.task.Scheduler.tick(Scheduler.java:88)
        at net.runelite.client.callback.Hooks.clientMainLoop(Hooks.java:89)
        at client.ax(client.java)
        at be.ab(be.java)
        at be.run(be.java)
        at java.lang.Thread.run(Thread.java:745)
Caused by: java.lang.NullPointerException
        at bj.getName(bj.java)
        at net.runelite.client.plugins.idlenotifier.IdleNotifierPlugin.checkIdle(IdleNotifierPlugin.java:200)
        ... 10 more
```

**Comment by UniquePassive** [comment]
getName() in the RSPlayerMixin calls getRSName(), which gets the value of Player#name, and then calls .replace on it, like so:
```java
@Inject
@Override
public String getName()
{
	return getRSName().replace('\u00A0', ' ');
}
```

I believe the fact that getRSName() (Player#name) can be null is what causes your NPE. 

If I look at the game's code, the name field is only null if the player's appearance hasn't been decoded and applied yet. I measured the time between the local player being set and its appearance being decoded and that took 160ms, so there's a window in which Player#name is null.

This should be fixed with https://github.com/runelite/runelite/pull/293.

**Comment by Adam-** [comment]
Fixed in #293
